cypress/page/index-demoqa.ts

```diff
@@ -1,8 +1,9 @@
 export { Login } from "./login-demoqa.page";
 export { PersonalInformation, UserInformation } from "./personal-information.page";
+//import "./commands";
```

Comment:
Remove line

Reply:
I don't see this line in the project

![image](https://user-images.githubusercontent.com/70541865/232544228-f43ac3dc-a635-4f19-83a0-fa4deb1e4662.png)

Reply:
update